src/components/Footer/Footer.scss

```
@@ -0,0 +1,153 @@
+.footer {
+  background-color: black;
+  .footerTop {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 3rem 0 3rem;
+
+    .footerTopLeftItem {
+      display: flex;
+      justify-content: center;
+
+      .newsList {
+        list-style: none;
+        padding-left: 0;
+        color: white;
+
+        li {
+          margin: 1rem;
+        }
+      }
+
+      .helpList {
+        list-style: none;
+        padding-left: 8rem;
+        font-size: 0.85rem;
+        color: gray;
+
+        li {
+          margin: 1rem;
+        }
+
+        & > li:nth-child(1) {
+          font-size: 1rem;
+          color: white;
+        }
+      }
+
+      .aboutList {
+        list-style: none;
+        padding-left: 8rem;
+        font-size: 0.85rem;
+        color: gray;
+
+        li {
+          margin: 1rem;
+        }
+
+        & > li:nth-child(1) {
+          font-size: 1rem;
+          color: white;
+        }
+      }
+    }
+
+    .footerTopRightItem {
+      display: flex;
+      justify-content: right;
+      font-size: 1.5rem;
+      color: white;
+      padding-top: 0.8rem;
+
+      .twitterIcon {
+        padding-right: 0.8rem;
+      }
+
+      .youtubeIcon {
+        padding-right: 0.8rem;
+      }
+    }
+  }
+
+  .footerCenter {
+    display: flex;
+    justify-content: space-between;
+    margin: 2.6rem 3rem 0 3rem;
+
+    .footerCenterLeftItem {
```

```suggestion
    &LeftItem {
```

footerCenter라는 컨텍스트가 유지되고 있기 때문에 가독성 측면에서 &LeftItem 으로 수정하면 더 좋을 것 같습니다!

---

수정 완료했습니다.